File: app/src/main/java/ru/eva/oriokslive/models/miet/news/NewsResponse.java
package ru.eva.oriokslive.models.miet.news;

import org.simpleframework.xml.Element;
import org.simpleframework.xml.Root;

@Root(name = "rss", strict = false)
public class NewsResponse {

    @Element(name = "channel")
    private Channel channel;

    public Channel getChannel() {
        return channel;
    }
}

File: app/src/main/java/ru/eva/oriokslive/network/entity/news/NewsResponse.kt
package ru.eva.oriokslive.network.entity.news

import org.simpleframework.xml.Element
import org.simpleframework.xml.Root

@Root(name = "rss", strict = false)
data class NewsResponse(
    @field:Element(name = "channel")
    @param:Element(name = "channel")
    val channel: Channel,
)